web/themes/custom/jcc_elevated/templates/paragraphs/paragraph--webform.html.twig

```diff
@@ -0,0 +1,28 @@
+{{ attach_library("jcc_storybook/Accordion") }}
+{{ attach_library("jcc_storybook/Container") }}
```

Comment:
Please remove Container and Stack

Reply:
I removed the template until a storybook component can be made